spec/vitest_spec.vim

```diff
@@ -18,7 +18,23 @@ describe "vitest"
   end
 
   context "on nearest tests"
-    it "runs JavaScript"
+    it "runs Javascript"
+      view +1 __tests__/normal-test.js
+      TestNearest
+
+      Expect g:test#last_command == g:expectedExecutable .. 'vitest run --no-coverage -t ''^Math'' __tests__/normal-test.js'
+
+      view +2 __tests__/normal-test.js
+      TestNearest
+
+      Expect g:test#last_command == g:expectedExecutable .. 'vitest run --no-coverage -t ''^Math Addition'' __tests__/normal-test.js'
+
+      view +3 __tests__/normal-test.js
+      TestNearest
+
+      Expect g:test#last_command == g:expectedExecutable .. 'vitest run --no-coverage -t ''^Math Addition adds two numbers$'' __tests__/normal-test.js'
+    end
+    it "runs JSX"
```

Comment:
Vitest supports Javsacript out of the box and the default configuration should also support JS. The previous author had mislabeled JSX extensions as "Javascript" and TSX extensions as "React" which is a bit odd